pkg/asset/installconfig/aws/metadata.go

```diff
@@ -134,22 +134,19 @@ func (m *Metadata) populateSubnets(ctx context.Context) error {
 	m.privateSubnets = sb.Private
 	m.publicSubnets = sb.Public
 	m.edgeSubnets = sb.Edge
-	return err
+	return nil
```

Comment:
If are going to return `nil` here, then we need to check `err` after calling `subnets`, otherwise we're just ignoring the errors.